src/Ledger/LedgerForm/utils.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { validateRequired } from '@folio/stripes-acq-components';
+
+export const validateLedger = async (fetchLedger, ledgerId, paramValue, paramName) => {
```

Comment:
it's not actually `utils`, it's more a service that calls API to validate. IMHO, since this file has one function, it can have the name of the function